lib/web/databases.go

```diff
@@ -141,16 +117,40 @@ func (h *Handler) handleDatabaseCreate(w http.ResponseWriter, r *http.Request, p
 
 // updateDatabaseRequest contains some updatable fields of a database resource.
 type updateDatabaseRequest struct {
-	CACert string `json:"ca_cert,omitempty"`
+	// CACert when non-nil means that this request is to
+	// only update the CA and validates it.
+	// When nil, skips validating CACert and updates all other
+	// fields.
+	CACert *string    `json:"ca_cert,omitempty"`
+	Labels []ui.Label `json:"labels,omitempty"`
+	URI    string     `json:"uri,omitempty"`
+	AWSRDS *awsRDS    `json:"awsRds,omitempty"`
```

Comment:
Can the JSON fields here be consistent? Either all in snake_case or camelCase

Reply:
camelCased `ca_cert`